src/Calculator/Views/MainPage.xaml.cs

```diff
@@ -15,6 +15,8 @@
 using Windows.UI.ViewManagement;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
```

Comment:
Could you remove these in TitleBar.xaml.cs if they are no longer used?

Reply:
done